conf/solr/7.7.2/schema.xml

```diff
@@ -38,7 +38,7 @@
     catchall "text" field, and use that for searching.
 -->
 
-<schema name="default-config" version="1.6">
+<schema name="default-config" version="1.7">
```

Comment:
When PR #7645 gets merged (in QA), this PR will need to be updated, as this edit is made to the soon-to-be older 7.7.2 schema.